### ISSUE TITLE
Wens om naam te geven: Portaal Wet Open Overheid Overijssel

### ISSUE THREAD

**Issue by gerardnienhuis** [IDgis/Onderzoeksbibliotheek#9]
Naam terug laten komen in de website. Naam wordt:

Portaal Wet Open Overheid Overijssel

**Comment by KevinvdBosch** [comment]
Moeten teksten op de home en contact pagina daar ook bij worden aangepast?

**Comment by gerardnienhuis** [comment]
ja, overal waar onderzoeksbibliotheek of onderzoeksbank staat, daar moet komen te staan: Portaal Wet Open Overheid Overijssel

**Comment by HermanAssink** [comment]
Teksten op Home en Contact-pagina zullen verschillen voor Onderzoeksbibliotheek en WOO-Portaal. Verder Moet de tweede tab "Onderzoeken" bij WOO-Portaal ook een andere naam krijgen. M.a.w. teksten moeten per omgeving gevarieerd kunnen worden.
